validation-test/compiler_crashers_2_fixed/0138-rdar36449760.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: not %target-swift-frontend %s -typecheck
+
+protocol A {
+  struct B {
+    var answer: Int = 42
+  }
+}
+
+class C : A {
+  func foo() -> B {}
+  func bar() -> A.B {}
+  func baz(b: B) {
+    _ = b.answer
```

Comment:
It'd be good to have these same tests nested (1) in a generic type, and (2) nested within yet another type inside C that does not conform to A.

Reply:
Sounds good, will do!